### PR TITLE
Use same version on windows & linux

### PR DESCRIPTION
**Pull request by kamaradclimber**
This patch also stop consul service before upgrading on windows

### CODE REVIEW

**Comment by kamaradclimber** [comment]
cc @Annih

**Comment by kamaradclimber** [comment]
cc @a.boten

File: recipes/install_windows.rb [hunk 1]
```diff
@@ -23,6 +23,23 @@
 
   include_recipe 'chocolatey::default'
 
+  # stop consul if we want to upgrade the version
+  service 'consul' do
+    action :stop
+    not_if do
```
Comment (Annih) [inline recipes/install_windows.rb]: You mean `only_if`, don't you?

**Comment by Annih** [comment]
LGTM

**Pull request merged**